tests/application/cms/test_views.py

```diff
@@ -808,7 +808,12 @@ def test_edit_measure_page_updated_with_latest_db_version_id_when_posting_a_conf
     assert page.find("input", {"id": "db_version_id"}).attrs["value"] == "2"
     assert measure_version.title == "new title"
     assert page_displays_error_matching_message(
-        response, message=f"has been updated by {measure_version.last_updated_by}"
+        response, message=f"‘Title’ has been updated by {measure_version.last_updated_by}"
+    )
+    assert len(page.findAll("div", class_="eff-diffs")) == 1
+    assert (
+        str(page.find("div", class_="eff-diffs"))
+        == '<div class="govuk-inset-text eff-diffs">\n<ins>try</ins> new title\n  </div>'
```

Comment:
_Maybe_ assert that there is an `ins` element containing the text `'try'`?

Reply:
As above. I quite like asserting this full-text string. I know it's brittle, but it's also relatively easy to update and locks in that we're getting the right styles and expected ins/del blocks.

Reply:
ok, let's keep.